setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+setup(
```

Comment:
change setup.py to install executable

Reply:
Add in e6bd4c559e8de5f1b2fe8b05faa47c8103c54e6f